stdlib/LinearAlgebra/src/lapack.jl

```diff
@@ -95,6 +95,23 @@ function chkfinite(A::AbstractMatrix)
     return true
 end
 
+function chkuplofinite(A::AbstractMatrix, uplo::AbstractChar)
```

Comment:
To play it safe, I think we should also check that we may indeed assume one-based indexing:
```suggestion
function chkuplofinite(A::AbstractMatrix, uplo::AbstractChar)
    require_one_based_indexing(A)
```

Reply:
... and once we have that, the loops below should get a `@inbounds` prepended.